en/xml/fo/topic.graphic-pageheader.xml

```diff
@@ -0,0 +1,259 @@
+                                pages on the right side.</para>
+                        </callout>
+                        <callout arearefs="co.topic.graphic-pageheader.height">
+                            <para>To avoid conflicts with the body text area,
+                                we limit its height to 1 inch. When you
+                                specify only one parameter like
+                                    <tag>height</tag>or<tag>width</tag>, your
+                                graphic will be scaled with the <link
+                                    xlink:href="#li.fo.external-graphics"
+                                    >correct aspect ratio</link>.</para>
+                        </callout>
+                        <callout
+                            arearefs="co.topic.graphic-pageheader.attribute">
+                            <para>The filename is passed on the <tag
+                                    class="attribute">src</tag> attribute.
+                                    <emphasis>src</emphasis>.</para>
+                        </callout>
+                        <callout arearefs="co.topic.graphic-pageheader.template">
+                            <para>The template which is called to import and
+                                place the graphic.</para>
+                        </callout>
+                        <callout arearefs="co.topic.graphic-pageheader.file">
+                            <para>Our file defined at the top of our
+                                customization layer.</para>
+                        </callout>
+                    </calloutlist>
+                </example>
+            </step>
+            <step>
+                <para>Use your customization layer to transform your DocBook
+                    document into FO.</para>
+            </step>
+        </procedure>
+    </section>
+    <section role="discussion">
+        <title>Discussion</title>
+        <para>These points will be submitted later</para>
+        <itemizedlist>
+            <listitem>
+                <para>Logo on recto/verso pages (double-sided docs)</para>
+            </listitem>
+            <listitem>
+                <para>Logo in the footer</para>
+            </listitem>
+            <listitem>
+                <para>Logo and chapter title in the page header</para>
+            </listitem>
+        </itemizedlist>
+    </section>
+    <section role="seealso">
+        <title>See Also</title>
+        <itemizedlist>
+            <listitem xml:id="li.fo.external-graphics">
+                <para>Full parameter description for
+                        <tag>fo:external-graphic</tag><link
+                        xlink:href="http://www.w3.org/TR/xsl/#fo_external-graphic"
+                    /></para>
+            </listitem>
+            <listitem>
+                <para>Parameter description for <tag>xsl:attribute</tag><link
```

Comment:
I'm not sure why you've add this, but IMHO this isn't really needed.

Reply:
Removed.